packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -57,6 +56,154 @@ class NetworkController extends DisposableController
     subscribeToFilterChanges();
   }
 
+  String? exportAsHarFile() {
+    httpRequests =
+        filteredData.value.whereType<DartIOHttpRequestData>().toList();
+
+    if (httpRequests!.isEmpty) {
+      debugPrint('No valid request data to export');
+      return '';
+    }
+
+    try {
+      if (httpRequests!.isNotEmpty) {
+        final har = {
+          'log': {
+            'version': '1.2',
+            'creator': {
+              'name': 'flutter_tool',
```

Comment:
Maybe this should be `dart_devtools` or `devtools`? Network profiling is supported for standalone Dart applications as well.

Reply:
changed to devtools